config/keycloak/realms/ovs-local-realm.json

```diff
@@ -988,25 +988,99 @@
       "enabled": true,
       "alwaysDisplayInConsole": false,
       "clientAuthenticatorType": "client-secret",
-      "redirectUris": [],
-      "webOrigins": [],
+      "secret": "odl-video-secret-2025",
+      "redirectUris": [
+        "http://ovs.odl.local:8089/auth/complete/keycloak/*",
+        "http://ovs.odl.local:8089/auth/complete/keycloak/",
+        "http://ovs.odl.local:8089/",
+        "http://localhost:8089/auth/complete/keycloak/*",
+        "http://localhost:8089/auth/complete/keycloak/",
+        "http://localhost:8089/"
+      ],
+      "webOrigins": [
+        "http://ovs.odl.local:8089",
+        "http://localhost:8089",
+        "http://localhost:8087"
+      ],
       "notBefore": 0,
-      "bearerOnly": true,
+      "bearerOnly": false,
       "consentRequired": false,
       "standardFlowEnabled": true,
       "implicitFlowEnabled": false,
-      "directAccessGrantsEnabled": false,
+      "directAccessGrantsEnabled": true,
       "serviceAccountsEnabled": false,
       "publicClient": false,
-      "frontchannelLogout": false,
+      "frontchannelLogout": true,
       "protocol": "openid-connect",
       "attributes": {
         "realm_client": "true",
         "post.logout.redirect.uris": "+"
       },
       "authenticationFlowBindingOverrides": {},
-      "fullScopeAllowed": false,
+      "fullScopeAllowed": true,
       "nodeReRegistrationTimeout": 0,
+      "protocolMappers": [
+        {
+          "id": "a1b2c3d4-e5f6-7890-abcd-ef1234567891",
```

Comment:
Keycloak fails to start properly because of the dupe id here:

>  org.hibernate.NonUniqueObjectException: A different object with the same identifier value was already associated with this persistence context for entity [org.keycloak.models.jpa.entities.ProtocolMapperEntity with id 'a1b2c3d4-e5f6-7890-abcd-ef1234567891']